Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:14
+
+# Create app directory
+WORKDIR /usr/src/app
```

Comment:
What are your thoughts on having a set DB to be loaded instead of whatever already exists locally? For example, this could include just the [example servers](https://github.com/mcode/medmorph-backend/wiki/Collections#examples).

Reply:
For development purposes, I suppose it could be useful to have some set of example data loaded into the database.  It's in-memory so we could write a JS file to just set it up and load data in, right?

Reply:
I was even just thinking a version of `medmorph.db` which the Dockerfile would copy over into `/usr/src/app/medmorph.db`

Reply:
ya that would also work, i don't see why not.

Reply:
This is partly why I wanted an easy to run script to add servers to the DB, so that we don't have to include a copy of the DB in the repo. Otherwise we'll have to remember to remove it when we want to deploy this somewhere

Reply:
that's also true, we could just run it to add a couple dummy servers in the dockerfile.